frontend/app/views/fragments/inlineIcon.scala.html

```diff
@@ -0,0 +1,5 @@
+@(name: String, classes: Seq[String] = List())
+
+<svg class="icon-inline @classes.map("" + _).mkString(" ")">
```

Comment:
TBH I used the same pattern from here https://github.com/guardian/frontend/blob/aa183a4395b1f89404585c85500471c083e2c036/common/app/views/fragments/inlineSvg.scala.html

I'm not appending a prefix here, so it feels a little awkward. I'm sure there's a better way of doing this. I just want to loop over an optional list of extra classes.